### DISCUSSION TITLE
IPXE  tftp Timeout

### DISCUSSION THREAD

**Discussion by lpzs** [ipxe/ipxe#633] [General]
Hi there,
I'm experiencing a random TFTP timeout when it downloads the boot.wim using "snponly.efi". We are using the latest ipxe build. haven't experienced this while using "lpxelinux.0". we do have multiple VLANs though.

**Comment by epacua** [comment]
Downloading a huge file like *.wim via tftp does not seem a good idea. Perhaps you can try http download. A packet capture will certainly helps in troubleshooting this issue.

**Comment by stappersg** [comment]
On Wed, Apr 20, 2022 at 11:19:16AM -0700, lpzs wrote:

> I'm experiencing a random TFTP timeout when it downloads the boot.wim using "snponly.efi".

And what reports the TFTP-time-out?


> We are using the latest ipxe build.

And when using a modified version that has increased time-out-time?


> haven't experienced this while using "lpxelinux.0".

And why is that a valid compare with boot.wim?


> we do have multiple VLANs though. 

And what is visible with tcpdump (or wireshark) on those VLANs
regarding the reported random TFTP timeout?
